packages/elements/src/icon/utils/IconLoader.ts

```diff
@@ -12,14 +12,21 @@ const iconLoaderInstance = new IconLoader();
 export { iconLoaderInstance as IconLoader };
 
 /**
- * @deprecated Icon `preload` method is deprecated.
+   * Deprecation noticed, used to display a warning message
+   * when deprecated features are used.
```

Comment:
```suggestion
   * Deprecation notice displays a warning message
   * when deprecated features are used.
```

there probably are other location with similar comment. Could you fix them as well?

Reply:
Updated 😉👍